vendor/github.com/Microsoft/opengcs/client/config.go

```diff
@@ -110,7 +110,7 @@ func ParseOptions(options []string) (Options, error) {
 		rOpts.Vhdx = filepath.Join(rOpts.KirdPath, `uvm.vhdx`)
 	}
 	if rOpts.KernelFile == "" {
-		rOpts.KernelFile = `bootx64.efi`
+		rOpts.KernelFile = `kernel`
```

Comment:
Will this cause issues for containers created with a version of Docker before this change?

Reply:
@thaJeztah That's why I gave @rn a heads up on this. He's aware :)

Reply:
no issue with containers created before this change. But Docker Desktop needs to  change how the file is named next time with update.

Reply:
Saw the heads up; but was wondering if it would affect existing containers as well :sweat_smile: